packages/main/src/Input.ts

```diff
@@ -780,6 +783,19 @@ class Input extends UI5Element implements SuggestionComponent, IFormElement {
 		} else {
 			this.closePopover();
 		}
+
+		if (this._performTextSelection) {
+			// modify DOM directly due to some issues with lit rendering
```

Comment:
Could you please write something more specific, as this comment is not that useful?